packages/next/package.json

```diff
@@ -25,7 +25,8 @@
 	"dependencies": {
 		"@prefresh/babel-plugin": "0.2.2",
 		"@prefresh/core": "^1.0.0",
-		"@prefresh/utils": "^1.0.0"
+		"@prefresh/utils": "^1.0.0",
+		"@prefresh/webpack": "^2.0.0"
```

Comment:
Would a changeset release set this to `^2.1.0` as this would be required to have the next.js support in `@prefresh/webpack`

Reply:
It probably would not since this has `^`